bundles/org.openhab.binding.radiothermostat/README.md

```diff
@@ -0,0 +1,201 @@
+# RadioThermostat Binding
+
+This binding connects RadioThermostat/3M Filtrete models CT30, CT50/3M50, CT80, etc. with built-in Wi-Fi module to openHAB.
+
+The binding retrieves and periodically updates all basic system information from the thermostat. The main thermostat functions such 
```

Comment:
Please add a line break after every sentence throughout the README, thanks!

Reply:
got it